lab1.js

```diff
@@ -133,7 +158,20 @@ var tooHungryDay;
  meals)
 */
 
-// your code goes here
+var average = 0;
+var sum = 0;
+var tooHungryDay;
+var sum;
+
+for (var i = 0; i < mealsPerDay.length ; i++)
+{
+  sum += mealsPerDay[i];
+  average = sum / (i + 1);
+  if (average < 4) {
+    tooHungryDay = i + 1;
+    //break; This made the assert fail.
```

Comment:
without having this break, tooHungryDay just keeps increasing - first 7, then 8, then 10. That's why the assert is failing. If you put the break back in, the assert passes.